evalml/pipelines/components/transformers/preprocessing/delayed_feature_transformer.py

```diff
@@ -44,6 +44,7 @@ def __init__(self, max_delay=2, delay_features=True, delay_target=True, gap=1,
 
     def fit(self, X, y=None):
         """Fits the DelayFeatureTransformer."""
+        return self
```

Comment:
All fit should return self, even if it doesn't do anything!

Reply:
Yup, I believe returning `self` is done in our `ComponentBase` class after some logic! By implementing fit here, I think that fit method was overridden by this one--which originally didn't do anything and didn't return `self` so was therefore inconsistent :)